lib/active_merchant/billing/gateways/quickpay/quickpay_v10.rb

```diff
@@ -15,7 +15,7 @@ def initialize(options = {})
       end
 
       def purchase(money, credit_card_or_reference, options = {})
-        MultiResponse.run(true) do |r|
+        process = MultiResponse.run(true) do |r|
```

Comment:
@curiousepic I believe if you don't pass `true` to the `MultiResponse.run` it'll default to using the last response as the primary response?

https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/response.rb#L44

I think that would avoid setting it to a variable then pulling the last response.